src/services/scan_form_service.js

```diff
@@ -8,17 +8,35 @@ export default (easypostClient) =>
 
     /**
      * Create a ScanForm.
-     * @param {object} prams
+     * @param {object} params
      * @returns {ScanForm}
      */
     static async create(params) {
-      try {
-        // TODO: We should re-implement the check here that wraps up params in `shipments` if the user didn't
-        const response = await easypostClient.post(this._url, params);
+      // TODO: We should re-implement the check here that wraps up params in `shipments` if the user didn't
```

Comment:
Do you mind tackling this simple TODO? If you go to the commit before mine, you'll find the previous implementation. It just needed a little tweak to properly wrap up the params. Makes sense to do this now since we are in here messing with the function implementation.

If not, I can do it after, but saves us all a review.